docs/News.rst

```diff
@@ -0,0 +1,59 @@
+.. _News:
+
+###############
+Creating a News
+###############
+
+From a space's acivity stream, you are able to share a news with the space’s members. 
+
+======================================
+Sharing a simple news from short form
+======================================
+
+To share an article with other space’s members, follow these steps:
+
+
+1. Click on News tab from the activity composer, The tab contains :
```

Comment:
IMO we can simplify by saying:
```
To share an article with other space’s members, click on the News tab in the activity composer and fill the fields:
- A **Title** field: ...
...
Then click on the Post button.
```
And only one image with the whle form is enough IMO. WDYT ?

Reply:
I updated it the message and put it in the preprod:
https://docs-preprod.exoplatform.org/en/preproduction/News.html
It is more simple like that. I agree with you

Reply:
I updated it the message and put it in the preprod:
https://docs-preprod.exoplatform.org/en/preproduction/News.html
It is more simple like that. I agree with you

Reply:
It is better indeed :+1: 
No need to add the image of the Post button alone after "Then type your News Title and content in the appropriate fields and click on Post button" IMO, it is clear that we talk about the Post button of the previous screenshot.
Also, the screenshot should be updated to remove the comment, to be in english, and to have the new logo.

Reply:
It is better indeed :+1: 
No need to add the image of the Post button alone after "Then type your News Title and content in the appropriate fields and click on Post button" IMO, it is clear that we talk about the Post button of the previous screenshot.
Also, the screenshot should be updated to remove the comment, to be in english, and to have the new logo.

Reply:
It is better indeed :+1: 
No need to add the image of the Post button alone after "Then type your News Title and content in the appropriate fields and click on Post button" IMO, it is clear that we talk about the Post button of the previous screenshot.
Also, the screenshot should be updated to remove the comment, to be in english, and to have the new logo.

Reply:
> No need to add the image of the Post button alone after "Then type your News Title and content in the appropriate fields

>  I followed the example in other documentation pages
![image](https://user-images.githubusercontent.com/45466574/59675879-94f2c100-91c6-11e9-82d2-c5e821d38b8a.png)

> Also, the screenshot should be updated to remove the comment, to be in english, and to have the new logo.

> 
Done
>